packages/arb-avm-cpp/avm_values/include/avm_values/value.hpp

```diff
@@ -18,10 +18,11 @@
 #define value_hpp
 
 #include <avm_values/bigint.hpp>
+#include <avm_values/opcodes.hpp>
 
 #include <nonstd/variant.hpp>
 
-enum ValueTypes { NUM, CODEPT, HASH_PRE_IMAGE, TUPLE };
+enum ValueTypes { NUM, CODEPT, HASH_PRE_IMAGE, TUPLE, CODE_POINT_STUB = 12 };
```

Comment:
seems to be a mismatch with the ethbridge side? where VALUE_TYPE_COUNT is 12

Reply:
This one is a bit awkward, but I added comments here and in the ethbridge to explain why